package.json

```diff
@@ -157,7 +157,7 @@
     "@angular-devkit/schematics": "^7.3.9",
     "@types/chai": "4.1.2",
     "@types/lodash": "4.14.102",
-    "@types/mocha": "2.2.48",
+    "@types/mocha": "5.0.0",
```

Comment:
Probably don't need to update mocha as part of this change.

Reply:
It's only a **types** update. 
The old types didn't allow passing a message to the `done` function indicating what caused the failure (not to mention the version mismatch we previously had).
Any reason not to make this change (other than separation of changes)?